e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+create_infra "$namespace"
+cluster="some-name"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running "$cluster-rs0" 3
+
+desc 'add labels and annotation manually'
+# `notIgnoredLabel` and `notIgnoredAnnotation` should be deleted
+kubectl_bin patch "service/$cluster-rs0-0" --type=json --patch '[
+    {
+        "op": "add", 
+        "path": "/metadata/labels", 
+        "value": {
+            "notIgnoredLabel": "true",
+            "ignoredLabel": "true"
+        }
+    },
+    {
+        "op": "add", 
+        "path": "/metadata/annotations", 
+        "value": {
+            "notIgnoredAnnotation": "true",
+            "ignoredAnnotation": "true"
+        }
+    }]'
+
+sleep 5 # waiting for reconcile
+desc 'check if not ignored annotations/labels are deleted from service'
+compare_kubectl "service/$cluster-rs0-0"
```

Comment:
@pooknull what about e.g. mongos? I think we need to test all services (expose sections).